### DISCUSSION TITLE
Bulk Check-out

### DISCUSSION THREAD

**Discussion by cfurr89** [snipe/snipe-it#13877] [Ideas]
I couldn't find if this has already been brought up, but I would like the ability to check/select a handful of assets and bulk-check them out without needing to scan or have the asset on hand. We recently swapped from a different software and are having to manually check out each individual asset that is already at a location. It would be much faster to be able to go to a location, select all assets, and then check them out to that location. OR, when updating status on multiple assets to checked out, it should reflect that the assets are checked out. This may just be a bug.

**Comment by snipe** [comment]
There is already internal discussion about the best way to handle this
